examples/azure-service-mesh/security/cert-manager-letsencrypt/readme.md

```diff
@@ -0,0 +1,276 @@
+# Cert-Manager Let's Encrypt Integration with Istio-based service mesh add-on
+
+This document contains instructions on how to integrate Istio-based service mesh add-on for AKS with cert-manager and obtain let's encrypt certificates for setting up secure ingress gateways. 
+
+## Objectives
+* Deploy bookinfo demo app, expose a secure HTTPS service using simple TLS.
+* Demonstrate HTTPS connections for Azure Service Mesh workloads using cert-manager and let's encrypt as the certificate authority.
+
+## Before you begin
+* [Install](https://learn.microsoft.com/en-us/azure/aks/istio-deploy-addon#install-istio-add-on) Istio-based service mesh add-on on your cluster.
+```shell
+az aks mesh enable -g <rg-name> -n <cluster-name>
+```
+* [Enable external ingressgateway](https://learn.microsoft.com/en-us/azure/aks/istio-deploy-ingress#enable-external-ingress-gateway)
+```shell
+az aks mesh enable-ingress-gateway -g <rg-name> -n <cluster-name> --ingress-gateway-type external
+```
+* [Enable sidecar injection](https://learn.microsoft.com/en-us/azure/aks/istio-deploy-addon#enable-sidecar-injection) on the default namespace. 
+```shell
+revision=$(az aks show --resource-group <rg-name> --name <cluster-name> --query 'serviceMeshProfile.istio.revisions[0]' -o tsv)
+kubectl label namespace default istio.io/rev=$revision
+```
+
+## Steps
+### 1. Setup DNS record
+Set up a DNS record for the `EXTERNAL-IP` address of the external ingressgateway service with your cloud provider. In this example, we are setting up the DNS record for `4.153.8.39` with `test.dev.azureservicemesh.io`.
```

Comment:
Let's link to relevant Azure docs for setting the DNS record?

Reply:
The customer may or may not use Azure DNS.  I think it is fair to assume readers know how to add a DNS record.